relevanceai/client/client.py

```diff
@@ -356,21 +294,15 @@ def Dataset(
     @track
     def ClusterOps(
         self,
-        alias: str,
-        model=None,
-        dataset_id: Optional[str] = None,
-        vector_fields: Optional[List[str]] = None,
-        cluster_field: str = "_cluster_",
+        model,
```

Comment:
This needs to default to None to allow for reloading ClusterOps objects. This should also have `alias` and `vector_fields`